mappings/net/minecraft/world/World.mapping

```diff
@@ -14,6 +15,8 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 	FIELD field_299 diskSize J
 	FIELD field_300 illuminatedAreas Ljava/util/List;
 	FIELD field_301 tickableEntities Ljava/util/List;
+	FIELD field_302 worldTreeSet Ljava/util/TreeSet;
```

Comment:
This still isn't very descriptive, I think I need to see what you're seeing in the bytecode, but I likely am unable to access until tonight.